# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Davidmp <medipas@gmail.com>, 2023
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Davidmp <medipas@gmail.com>, 2023\n"
"Language-Team: Catalan (https://app.transifex.com/calamares/teams/20061/ca/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ca\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "S'instal·la el carregador d'arrencada."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""
"No s'ha pogut instal·lar el grub. No s'han definit particions a "
"l'emmagatzematge global."

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr "Error d'instal·lació del carregador d'arrencada"

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"No s'ha pogut instal·lar el carregador d'arrencada. L'ordre d'instal·lació "
"<pre>{!s}</pre>ha retornat el codi d'error {!s}."

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "No es pot escriure el fitxer de configuració de l'LXDM."

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "El fitxer de configuració de l'LXDM {!s} no existeix."

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "No es pot escriure el fitxer de configuració del LightDM."

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "El fitxer de configuració del LightDM {!s} no existeix."

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "No es pot configurar el LightDM."

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "No hi ha benvinguda instal·lada per al LightDM."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "No es pot escriure el fitxer de configuració de l'SLIM."

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr "El fitxer de configuració de l'SLIM {!s} no existeix."

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr ""
"No hi ha cap gestor de pantalla seleccionat per al mòdul displaymanager."

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"La llista de gestors de pantalla és buida o no definida ni a globalstorage "
"ni a displaymanager.conf."

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "La configuració del gestor de pantalla no era completa."

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Es creen initramfs amb dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr "Ha fallat executar dracut."

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""
"Dracut no s'ha pogut executar a l'objectiu amb el codi de retorn  "
"{return_code}"

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Tasca de python fictícia."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Pas de python fitctici {}"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "S'escriu fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Error de configuració"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "No s'han definit particions perquè les usi <pre>{!s}</pre>."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""
"No s'ha proporcionat el punt de muntatge perquè l'usi <pre>{!s}</pre>."

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""
"No hi ha cap configuració de <pre>{!s}</pre> perquè la usi <pre>{!s}</pre>."

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Configura el GRUB."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "S'estableix el rellotge del maquinari."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Es configura mkinitcpio."

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr "No s'han definit particions per a <pre>initcpiocfg</pre>."

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr "No hi ha punt de muntatge per a <pre>initcpiocfg</pre>."

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Es configuren initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Es configuren les llengües."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Es creen initramfs amb mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Ha fallat executar mkinitfs a la destinació."

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "El codi de sortida ha estat {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Es munten les particions."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr "Error intern en muntar els conjunts de dades zfs"

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr "No s'ha pogut importar zpool."

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr "No s'ha pogut desblocar zpool."

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr "No s'ha pogut establir el punt de muntatge de zfs."

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr "error de muntatge de zfs"

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Es desa la configuració de la xarxa."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Es configura el sevei OpenRC dmcrypt."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Instal·la els paquets."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Es processen paquets (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "S'instal·la un paquet."
msgstr[1] "S'instal·len %(num)d paquets."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Se suprimeix un paquet."
msgstr[1] "Se suprimeixen %(num)d paquets."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr "Error del gestor de paquets"

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""
"El gestor de paquets no ha pogut preparar les actualitzacions. "
"L'ordre<pre>{!s}</pre> ha retornat el codi d'error {!s}."

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""
"El gestor de paquets no ha pogut actualitzar el sistema. L'ordre "
"<pre>{!s}</pre> ha retornat el codi d'error {!s}."

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""
"El gestor de paquets no ha pogut fer canvis al sistema instal·lat. L'ordre "
"<pre>{!s}</pre> ha retornat el codi d'error {!s}."

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Configura el tema del Plymouth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "S'instal·len dades."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Configura els serveis d'OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "No es pot afegir el servei {name!s} al nivell d'execució {level!s}."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""
"No es pot suprimir el servei {name!s} del nivell d'execució {level!s}."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Servei - acció desconeguda <code>{arg!s}</code> per al servei {name!s} al "
"nivell d'execució {level!s}."

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "No es pot modificar el servei."

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"La crida de <code>rc-update {arg!s}</code> a chroot ha retornat el codi "
"d'error {num!s}."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "El nivell d'execució de destinació no existeix."

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"El camí per al nivell d'execució {level!s} és <code>{path!s}</code>, però no"
" existeix."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "El servei de destinació no existeix."

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"El camí per al servei {name!s} és <code>{path!s}</code>, però no existeix."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr "Configura les unitats del systemd"

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr "No es pot modificar la unitat."

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""
"La crida de <code>systemctl {_action!s}</code> a chroot ha tonat el codi "
"d'error {_exit_code!s}."

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr "L'acció de {_action!s} la unitat de systemd <code>{_name!s}</code>."

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "S'omplen els sistemes de fitxers."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "Ha fallat rsync amb el codi d'error {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Es desempaqueta la imatge {}/{}, fitxer {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Es comença a desempaquetar {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Ha fallat desempaquetar la imatge \"{}\"."

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "No hi ha punt de muntatge per a la partició d'arrel."

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr "globalstorage no conté una clau \"rootMountPoint\"."

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Punt de muntatge incorrecte per a la partició d'arrel"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr "El punt de muntage d'arrel és \"{}\", però no existeix."

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr "Configuració d'unpackfs incorrecta"

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr "No hi ha informació de configuració."

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "El sistema de fitxers per a {} ({}) no és admès pel nucli actual."

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "El sistema de fitxers font \"{}\" no existeix."

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""
"No s'ha pogut trobar unsquashfs, assegureu-vos que tingueu instal·lat el "
"paquet squashfs-tools."

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "La destinació \"{}\" al sistema de destinació no és un directori."

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr "La còpia de zfs ha generat un hostid."
